velog/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./reset.css" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>켈로그</title>
+    <script
+      src="https://kit.fontawesome.com/ad696c8593.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <!-- 로고 제목으로 대체 -->
+      <h1 class="title">켈로그</h1>
+      <!-- 우측 nav bar -->
+      <nav class="header_menu-bar">
+        <i class="fa-solid fa-moon fa-xl"></i>
+        <i class="fa-solid fa-magnifying-glass fa-xl"></i>
+        <a href="#">새 글 작성</a>
+      </nav>
+    </header>
+    <main>
+      <!-- 메인 nav bar -->
+      <nav class="main_menu-bar">
+        <section class="main_menu-bar-left">
+          <article>
+            <i class="fa-solid fa-arrow-trend-up fa-lg"></i>
+            <span>트렌딩</span>
+          </article>
+          <article>
+            <i class="fa-regular fa-clock fa-lg"></i>
+            <span>최신</span>
+          </article>
+        </section>
+        <section class="main_menu-bar-right"></section>
+      </nav>
+      <!-- card section -->
+      <section class="cards">
+        <article class="card">
+          <img src="./1657840_1.jpg" alt="포스트 썸네일" class="post-image" />
+          <section class="post-info">
+            <h2 class="post-title">켈로그 맛있게 먹는 방법</h2>
+            <p class="post-text">켈로그는 눅눅하게 먹어야 맛있다.</p>
+            <p class="post-time">5일 전 * 3개의 댓글</p>
```

Comment:
만약에 bem쓴다면
```suggestion
          <section class="post-info">
            <h2 class="post-info__title">켈로그 맛있게 먹는 방법</h2>
            <p class="post-info__text">켈로그는 눅눅하게 먹어야 맛있다.</p>
            <p class="post-info__time">5일 전 * 3개의 댓글</p>
```
정도로 데게찌!?

Reply:
맨날 styled-components나 emotion 사용하니깐 이런 클래스명 네이밍에 익숙하지 않네....
이해했다 감사!!